src/main/resources/acrolinxPluginScript.js

```diff
@@ -89,17 +89,14 @@ window.acrolinxPlugin =
   openLogFile: function(){
     openLogFileP();
   },
-  parseAllClusters: function(clusters) {
-    reuseTestFunction(clusters);
-  },
   onReusePrefixSearchResult: function(result) {
-    onReusePrefixSearchResultP(result);
+    onLivePrefixSearchResultP(result);
   },
   onReusePrefixSearchFailed: function() {
```

Comment:
The sidebar gives back the prefix / search string when calling this function. In case you want to make use of this.
`onReusePrefixSearchFailed(prefix: String): void;`

I will also update the interfaces name from sidebar side to live instead of reuse

Reply:
Will that be soon? I only changed the interal  onLivePrefixSearchResultP and left onReusePrefixSearchResult which is being accessed by the sidebar

Reply:
done 👍🏻